README.md

```diff
@@ -34,14 +34,15 @@ npm install
 npm run dev
 ```
 
-The default UI address is `http://localhost:8080`.
+The default UI address is `http://localhost:3000`.
```

Comment:
@Fine0830 Please confirm. Should we get this on 8080 to align with webspp deployment mode.

Reply:
It's okay that the port is `3000` in `dev` mode.